# Confirms to the Descriptor type in flirtlib
# For now, we only allow the beta grid descriptor

Vector[] hist
Vector[] variance
Vector[] hit
Vector[] miss
